linux_os/guide/system/network/networkmanager/networkmanager_dns_mode/ansible/shared.yml

```diff
@@ -0,0 +1,14 @@
+# platform = multi_platform_all
+# reboot = false
+# strategy = configure
+# complexity = low
+# disruption = low
+
+{{{ ansible_instantiate_variables("var_networkmanager_dns_mode") }}}
+
+{{{ ansible_ini_file_set("/etc/NetworkManager/NetworkManager.conf", "main", "dns", "{{ networkmanager_dns_mode }}") }}}
```

Comment:
```
3851 fatal: [192.168.124.141]: FAILED! => {
3852     "msg": "The task includes an option with an undefined variable. The error was: 'networkmanager_dns_mode' is        undefined. 'networkmanager_dns_mode' is undefined\n\nThe error appears to be in '/home/jcerny/work/git/scap-           security-guide/tests/logs/rule-custom-2023-10-02-1036/xccdf_org.ssgproject.content_rule_networkmanager_dns_mode.       yml': line 45, column 7, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe               offending line appears to be:\n\n\n    - name: Set 'dns' to '{{ networkmanager_dns_mode }}' in the [main] section      of '/etc/NetworkManager/NetworkManager.conf'\n      ^ here\nWe could be wrong, but this one looks like it might        be an issue with\nmissing quotes. Always quote template expression brackets when they\nstart a value. For              instance:\n\n    with_items:\n      - {{ foo }}\n\nShould be written as:\n\n    with_items:\n      - \"{{ foo          }}\"\n"
3853 }
```